src/utils/markdown.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { ReactMarkdown } from 'react-markdown';
+
+export const RenderMarkdown = ({ content }) => {
```

Comment:
@Moe-Shoman This is the component definition that's broken

Reply:
You did everything right 👍

Reply:
@elimisteve just didn't import `ReactMarkdown` correctly

Reply:
remove the curly braces and everything should work fine